### PR TITLE
Change nullable_varchar_vector to use similar representation to words.

### PR DESCRIPTION
**Pull request by wmeddie**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -164,8 +163,8 @@ CMake / parallelExecution := false
 VectorEngine / parallelExecution := false
 inConfig(VectorEngine)(Defaults.testTasks)
 def veFilter(name: String): Boolean = name.startsWith("com.nec.ve")
-VectorEngine / fork := true
-VectorEngine / run / fork := true
+VectorEngine / fork := false
```
Comment (wgip) [inline build.sbt:166]: Any reason to switch back to non-forking?

File: tests/tpchbench/run_ve.sh [hunk 2]
```diff
@@ -5,19 +5,21 @@ export SPARK_HOME=/opt/spark
 time $SPARK_HOME/bin/spark-submit \
     --master yarn \
     --num-executors=8 --executor-cores=1 --executor-memory=8G \
-    --deploy-mode client \
+    --deploy-mode cluster \
```
Comment (wgip) [inline tests/tpchbench/run_ve.sh:8]: what is the difference between the two here?

**Comment by wgip** [comment]
@wmeddie - did a run on on this branch, is the output expected as per the run? https://github.com/XpressAI/SparkCyclone/runs/4628808040?check_suite_focus=true

**Comment by wmeddie** [comment]
Still needed but we need to re-do this PR from the latest code.

**Pull request closed without merging**